files/image_config/misc/docker-wait-any

```diff
@@ -14,39 +14,108 @@
     NOTE: This script is written against docker Python package 4.1.0. Newer
     versions of docker may have a different API.
 """
-
+import argparse
+import subprocess
 import sys
 import threading
+import time
+
 from docker import APIClient
+from sonic_py_common import logger
+from swsssdk import SonicV2Connector
+
+SYSLOG_IDENTIFIER = 'docker-wait-any'
+
+# Global logger instance
+log = logger.Logger(SYSLOG_IDENTIFIER)
 
 # Instantiate a global event to share among our threads
 g_thread_exit_event = threading.Event()
+g_service = []
+g_dep_services = []
+
+# Check if System warm reboot, or Container warm restart is enabled.
+def is_warm_restart_enabled(container_name):
+    state_db = SonicV2Connector(host='127.0.0.1')
```

Comment:
@judyjoseph I am wondering should we move this function to some common place as this looks to be generic and can be used by any service/application if needed.

Reply:
I could move this to sonic_py_common/device_info.py -- if that is a good place

Reply:
taken care of